types/adeunis/ftdNetwokTester/uplink.js

```diff
@@ -60,14 +61,15 @@ function consume(event) {
       const lonMin01 = bytes[offset + 6] & 0x0f;
       const lonMin001 = bytes[offset + 7] >> 4;
       const lonSign = bytes[offset + 7] & 0x01 ? -1 : 1;
-      decoded.longitude =
+      sample.longitude =
         lonSign *
         (lonDeg100 * 100 +
           lonDeg10 * 10 +
           lonDeg1 +
           (lonMin10 * 10 + lonMin1 + lonMin01 * 0.1 + lonMin001 * 0.01) / 60);
-      decoded.altitude = 0; // altitude information not available
-      decoded.sats = bytes[offset + 8] & 0x0f; // number of satellites
+      sample.sats = bytes[offset + 8] & 0x0f; // number of satellites
```

Comment:
rename variable sample and decoded to be meaningful and indicating what they are used for